website/docs/Conditions.md

```diff
@@ -0,0 +1,166 @@
+      value: "0"
+    }
+  }
+```
+
+### Own at least one ERC20 Token
+Here we're checking whether the user owns any `T` Threshold Network token
+```js
+const ERC20Conditions = {
+    contractAddress: "0xCdF7028ceAB81fA0C6971208e83fa7872994beE5",
+    standardContractType: "ERC20",
+    chain:  "ethereum",
+    method: "blanceOf",
+    parameters: [
+      ":userAddress"
+    ],
+    returnValueTest: {
+      comparator: ">",
+      value: "0"
+    }
+  }
+```
+
+### Ownership of at least one ERC1155 token from a batch of ids
+Batching can be applied to ERC721 tokens as well.
+```js
+const ERC115Conditions = {
+    contractAddress: "0x54F456B544abFb785694400bcb1D85629B2D437f",
+    standardContractType: "ERC115",
+    chain:  "ethereum",
+    method: "blanceOfBatch",
+    parameters: [
+      ":userAddress,:userAddress,:userAddress,:userAddress",
+      "1,2,1001,1002"
+    ],
+    returnValueTest: {
+      comparator: ">",
+      value: "0"
+    }
+  }
+```
+
+### Function call of non standard Contract
+In this example we will check that the user is staking `T` Threshold Token.
+The Threshold staking contract is located at [`0x01B67b1194C75264d06F808A921228a95C765dd7`](https://etherscan.io/address/0x01b67b1194c75264d06f808a921228a95c765dd7#readProxyContract).
+The function we wish to call is `stakes` which takes an `address` as it's parameter.
+We need to provide the `contractAddress`, `functionName`, `functionParams`, and `functionAbi` when defining the condition.
+
+```js
+const customABICondition = {
+    contractAddress: "00x01B67b1194C75264d06F808A921228a95C765dd7",
+    functionName: "stakes",
+    functionParams: [":userAddress"],
+    functionAbi: {
+        inputs: [
+        {
+            internalType: "address",
+            name: "stakingProvider",
+            type: "address"
+        }
+        ],
+        name: "stakes",
+        outputs: [
+        {
+            internalType: "uint96",
+            name: "tStake",
+            type: "uint96"
+        },
+        {
+            internalType: "uint96",
+            name: "keepInTStake",
+            type: "uint96"
+        },
+        {
+            internalType: "uint96",
+            name: "nuInTStake",
+            type: "uint96"
+        }
+        ],
+        stateMutability: "view",
+        type: "function"
+    },
+    chain:  "ethereum",
+    returnValueTest: {
+      key: "tStake",
+      comparator: ">",
+      value: "0"
+    }
+  }
+```
+
+## RPC Conditions
+
+Here will will query the ETH balance of the users address using the RPC call [`eth_getBalance`](https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_getbalance)
+```js
+const ETHBalance = {
+    contractAddress: "",
+    standardContractType: "",
+    chain:  "ethereum",
+    method: "eth_getBalance",
+    parameters: [
+      ":userAddress",
+      "latest"
+    ],
+    returnValueTest: {
+      comparator: ">=",
+      value: "10000000000000"
+    }
+  }
+```
+
+## Time Conditions
```

Comment:
I guess we are waiting to write this up until the code is changed to use blockTime instead of system time?

Reply:
exactly